cogs5e/models/sheet/player.py

```diff
@@ -168,7 +172,18 @@ def new(cls, character, name, minv=None, maxv=None, reset=None, display_type=Non
         elif max_value is not None:
             initial_value = max_value
 
-        return cls(character, name.strip(), initial_value, minv, maxv, reset, display_type, live_id, reset_to, reset_by)
+        # length checks
+        if desc and len(desc) > 800:
+            raise InvalidArgument('Description must be less than 800 characters.')
+
+        if title and len(title) >= 256:
+            raise InvalidArgument('Title must be less than 256 characters.')
+
+        if len(name) > 256:
+            raise InvalidArgument('Name must be less than 256 characters.')
+
+        return cls(character, name.strip(), initial_value, minv, maxv, reset, display_type, live_id,
+                   title, desc, reset_to, reset_by)
```

Comment:
```suggestion
                   reset_to, reset_by, title, desc)
```
to match the positional arg suggestions